frontend/sw360-portlet/src/main/resources/META-INF/resources/js/modules/sw360Wizard.js

```diff
@@ -178,7 +178,12 @@ define('modules/sw360Wizard', [ 'jquery', 'modules/button' ], function($, button
                     button.finish($('.wizardNext', $wizardRoot));
                 }).done(function(data, textStatus, xhr) {
                     try {
-                        var dataJson = JSON.parse(data);
+                        var dataJson = data;
+                        if (typeof data === 'string' || data instanceof String)
```

Comment:
only `typeof` check should do. Do we need to add `instanceof` check also ?

Reply:
Removed instanceof check. typeof check will serve the purpose